### ISSUE TITLE
antew/vim-elm-language-server merged into w0rp/ale

### ISSUE THREAD

**Issue by andys8** [kornicameister/dotfiles#206]
Once https://github.com/w0rp/ale/issues/2592 is merged and the plugin updated, `antew/vim-elm-language-server` can be replaced. The linter name is `elm_ls`. See https://github.com/elm-tooling/elm-language-server/pull/80/files for changes.

**Comment by kornicameister** [comment]
@andys8  thx for letting me know :+1: 
That's really nice of you :).
I will update my plugins once that's done :)

**Comment by kornicameister** [comment]
@andys8 is it time to make changes? Is everything completely merged?

**Comment by andys8** [comment]
Should be. Could you try it and confirm if it's working for you?

**Comment by kornicameister** [comment]
@andys8 it took longer then expected. Would you mind pointing me at a solution that would deal with en error coming from `deoplete` about missing `elm-oracle`.

**Comment by andys8** [comment]
Right

**Comment by kornicameister** [comment]
@andys8 something is not working.

There's a PR linked to this issue and given that setup some things stopped working (like for example there were obvious compilation errors not being highlighted by a plugin).

Now, I don't want to say that there's a bug in LSP perhaps my setup is not working.

Do you want to continue this discussion here or should I take my config and head to `elm-language-server` repository?

**Comment by andys8** [comment]
Would make sense to either open an issue in the elm-language-server repo, if you think there might be something wrong, or ask for help in #elm-language-server in the elm slack with more details :)

**Comment by kornicameister** [comment]
oh, so there's a channel in there :), I will certainly go there ;)
